MAINT: update munging

stsievert opened the pull request:
This updates the data munging from the notes in the notebook. Let me know if you have any questions.

jbednar commented on examples/Carbon_Flux.ipynb:
```diff
@@ -67,10 +67,14 @@
-    "        \n",
-    "    i = [col in df.columns for col in drop]\n",
+    "    \n",
+    "    assert all([col in df.columns for col in drop])\n",
```
Can you add a comment explaining what this bit does?

jlstevens commented on examples/Carbon_Flux.ipynb:
```diff
@@ -158,7 +200,7 @@
    "source": [
     "import scipy.io\n",
-    "rsif = scipy.io.loadmat('RSIF_2007_2016_05N_01L.mat')"
+    "rsif = scipy.io.loadmat('./flux_data/RSIF_2007_2016_05N_01L.mat')"
```
I note you've changed where to expect this `.mat` file. Either option is fine though this way means overwriting the git lfs 'file' of the same name...

stsievert commented:
I have updated this notebook.

* I have implemented a (very) simple linear model for one site (and do not include lat/lon as features).
* I have responded to the reviews (removed the `assert` and moved the .mat file)

I have also verified that the weights of the linear model vary based on location:


![features_trim](https://user-images.githubusercontent.com/1320475/43416624-5d505dee-9428-11e8-8ee2-61f053c7a220.png)


(this is data is trimmed a bit to remove certain sites and a couple features)

- [ ] TODO: I still need to perform bi-weekly averaging of RSIF data sampled at sites.
